system/dummy_diag_publisher/README.md

```diff
@@ -16,19 +16,38 @@ This package outputs a dummy diagnostic data for debugging and developing.
 
 ### Node Parameters
 
-| Name          | Type                           | Default Value | Explanation                             | Reconfigurable |
-| ------------- | ------------------------------ | ------------- | --------------------------------------- | -------------- |
-| `update_rate` | int                            | `10`          | Timer callback period [Hz]              | false          |
-| `diag_name`   | string                         | `diag_name`   | Diag_name set by dummy diag publisher   | false          |
-| `is_active`   | bool                           | `true`        | Force update or not                     | true           |
-| `status`      | DummyDiagPublisherNode::Status | `0 (OK)`      | diag status set by dummy diag publisher | true           |
+The parameter `DIAGNOSTIC_NAME` must be a name that exists in the parameter YAML file. If the parameter `status` is given from a commandline, the parameter `is_active` is automatically set to `true`.
```

Comment:
@asana17 Spell check failed in warning. 
> Warning: Unknown word (commandline)
This is not required, but highly recommended.